cmds/import_image/import_image_cmd_test.go

```diff
@@ -153,5 +160,72 @@ var _ = Describe("import-image command", func() {
 			Expect(importImageCmd.Uuid).ToNot(Equal(""))
 			Expect(importImageCmd.Id).ToNot(Equal(""))
 		})
+
+		It("creates a public VGDTG with UUID and ID - Positive path", func() {
+			fileNames := []string{
+				"SoftLayer_Virtual_Guest_Block_Device_Template_Group_Service_createFromExternalSource.json",
+				"SoftLayer_Virtual_Guest_Block_Device_Template_Group_Service_createPublicArchiveTransaction.json",
+			}
+			testhelpers.SetTestFixturesForFakeSoftLayerClient(fakeClient, fileNames)
+
+			options = common.Options{
+				NameFlag:       "fake-name",
+				NoteFlag:       "fake-note",
+				PublicFlag:     true,
+				PublicNameFlag: "fake-public-name",
+				PublicNoteFlag: "fake-public-note",
+				OsRefCodeFlag:  "fake-os-ref-code",
+				UriFlag:        "fake-uri",
+			}
+
+			importImageCmd, err = NewImportImageCmd(options, fakeClient)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(importImageCmd).ToNot(BeNil())
+
+			cmd = importImageCmd
+			common.TIMEOUT = 10 * time.Millisecond
+			common.POLLING_INTERVAL = 1 * time.Millisecond
+
+			err = cmd.Run()
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(importImageCmd.Uuid).To(Equal(""))
+			Expect(importImageCmd.Id).ToNot(Equal(""))
+		})
+		It("creates a public VGDTG with UUID and ID - Nagtive path", func() {
```

Comment:
remove `"Nagtive path"` (type in first word) and move this to a `Context("...")`

`Context` are meant exactly to divide tests into logical sections.